webapp/platform/client/src/websocket.ts

```diff
@@ -156,6 +161,27 @@ export default class WebSocketClient {
                 this.firstConnectListeners.forEach((listener) => listener());
             }
 
+            var waitingForPong = false;
+            this.pingInterval = setInterval(
```

Comment:
To be safe, could you check if `pingInterval` is already set here and clear it if it is? I don't know if that's actually a possible scenario, but I think it'd be good to play it safe here since this could cause issues for both client and server if we end up with multiple intervals running